shared/tracker/action.render.desktop.js

```diff
@@ -27,10 +28,16 @@ export default class ActionRender extends Component {
     }
   }
 
-  renderPending (): ReactElement {
-    return (
-      <div><p> Loading... </p></div>
-    )
+  renderPending (username: ?string): ReactElement {
+    const centered = {textAlign: 'center', paddingTop: 8}
+    if (username) {
+      return (
+        <div><Text style={centered} type='Body'>{`Verifying ${username}'s identity`}</Text></div>
```

Comment:
Also we should lean more towards the stateless components and avoid the methods I think (if there's literally no state, like this)